src/screens/Learning.js

```diff
@@ -122,6 +130,47 @@ export default ({
   const categoryHeading = LANG_DATA[CATEGORY_HEADING][nativeLanguage];
   const phraseHeading = LANG_DATA[PHRASE_HEADING][nativeLanguage];
   const solutionHeading = LANG_DATA[SOLUTION_HEADING][nativeLanguage];
+  const currentPhraseCategoryName = categories.find(category =>
+    category.phrasesIds.find(phraseId => phraseId === currentPhrase?.id),
+  );
+
+  const getCategoryName = () => {
+    const catNameInEnglish = `Learnt phrases - ${
+      currentPhraseCategoryName?.name[LANGUAGE_NAMES.EN]
+    }`;
+    const catNameInMalagasy = `Fehezanteny efa nianarana - ${
+      currentPhraseCategoryName?.name[LANGUAGE_NAMES.MG]
+    }`;
+
+    // Get category name function from utils
+    return getCurrentCategoryName(
+      currentCategoryName,
+      isLearntPhrases,
+      catNameInEnglish,
+      catNameInMalagasy,
+    );
+  };
+
+  const currentPhraseCategoryName = categories.find(category =>
+    category.phrasesIds.find(phraseId => phraseId === currentPhrase?.id),
+  );
```

Comment:
how did you managed to compile this? 
currentPhraseCategoryName definition is duplicated ?